pshtt.py

```diff
@@ -349,6 +511,12 @@ def is_hsts_preloaded(domain):
     return domain in preload_list
 
 
+# For "x.y.domain.gov", return "domain.gov".
+# TODO: use Public Suffix list to do this properly.
+def parent_domain_for(hostname):
+    return str.join(".", hostname.split(".")[-2:])
```

Comment:
Noting here that this logic is naive, and that to properly detect the "registerable" parent domain of a hostname requires using the Public Suffix list. This is not necessary for `.gov`, but for the tool to be generically useful, it will be required. I described this in #8.